### PR TITLE
Add codelabs to filter target

### LINKED ISSUE #387: Add Codelabs to filter target
## Kind (Required)

- Other

 ## Overview (Required)

- Currently, Codelabs is out of filter targets

- Add isFilterable property to [SessionType](https://github.com/DroidKaigi/conference-app-2019/blob/master/model/src/commonMain/kotlin/SessionType.kt) 
- Use the isFilterable property to determine if it is a filter target.

 ## Links	

- #231 
- #358

### PR DESCRIPTION
**Pull request by kirimin**
## Issue
- close #387

## Overview (Required)
- Added isFavoritable propaty to SessionType and use it to filering
- Fixed Tests.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Probably your master is not fetched 👀 
image

**Comment by kirimin** [comment]
Sorry for mistake. ><
I re-feched!

File: model/src/jvmTest/kotlin/io/github/droidkaigi/confsched2019/model/FiltersTest.kt [hunk 1]
```diff
@@ -11,8 +11,13 @@ class FiltersTest {
         assertTrue { Filters().isPass(mockk<Session.SpeechSession>()) }
     }
 
-    @Test fun isPass_WhenServiceSession() {
-        assertTrue { Filters().isPass(mockk<Session.ServiceSession>()) }
+    @Test fun isPass_WhenServiceSessionAndIsNotFilterable() {
```
Comment (takahirom) [inline model/src/jvmTest/kotlin/io/github/droidkaigi/confsched2019/model/FiltersTest.kt]: Thank you for testing 💯

**Comment by takahirom** [comment]
LGTM 🥇

**Pull request merged**